src/main/telemetry/hott.c

```diff
@@ -324,78 +330,19 @@ static void workAroundForHottTelemetryOnUsart(serialPort_t *instance, portMode_e
 {
     closeSerialPort(hottPort);
 
-    portOptions_e portOptions = telemetryConfig()->telemetry_inverted ? SERIAL_INVERTED : SERIAL_NOT_INVERTED;
```

Comment:
removing this will break devices that have an always-on hardware inverter on the telemetry port.